packages/material-ui/test/umd/run.js

```diff
@@ -47,7 +47,7 @@ async function createApp() {
 
   let index = await fse.readFile(path.join(rootPath, 'examples/cdn/index.html'), 'utf8');
   index = index.replace(
-    'https://unpkg.com/@material-ui/core@latest/umd/material-ui.development.js',
+    'https://unpkg.com/@material-ui/core@next/umd/material-ui.development.js',
```

Comment:
how is this related?

Reply:
The `replace` was doing nothing, because the path was using the `next` tag in `examples/cdn/index.html`. Since I had removed `createMuiTheme` it [broke](https://app.circleci.com/pipelines/github/mui-org/material-ui/41135/workflows/bd1eeab9-1ddd-43a0-a8e9-e5c0e5bf8f9d/jobs/245237) the e2e tests. Now that we have `createMuiTheme` back it could be reverted, or we leave it but once v5 becomes stable we have to update it back to `latest`.

https://github.com/mui-org/material-ui/blob/d7c0ae5958a274c6d7043b3bda8291a1d93fd3c1/examples/cdn/index.html#L9

Reply:
Got it, thanks! Let’s leave it I’d say

Reply:
I would personally keep latest so we don't forget to update it. On this CDN aspect, I was also eager to replace/complement this example with something more modern https://trello.com/c/8QGmlDLH